YoCode/Program.cs

```diff
@@ -41,6 +41,7 @@ private static void Main(string[] args)
             if (result.helpAsked)
             {
                 compositeOutput.ShowHelp();
+                htmlReportLaunched = HtmlReportLauncher.LaunchReport("YoCodeReport.html");
```

Comment:
This line happens in two places now, it would be good to only have it once. Maybe just extract a method called `ShowHtmlReport` that contains this one line? I don't think the `htmlReportLaunched` field is needed either, so you could get rid of that too.